__tests__/RaceToZero.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { SimulatorState } from "../src/lib/SimulatorState"
+import { RaceToZero } from "../src/lib/VictoryConditon"
+import { SimulatorStateFactory } from "./TestHelpers"
+
+describe("A race to zero victory condition", () => {
```

Comment:
Started defining unit tests for our victory conditions. Should have started doing this earlier TBH